Ginger/GingerCoreNET/WorkSpaceLib/Solution.cs

```diff
@@ -86,7 +86,7 @@ public void SaveSolution(bool showWarning = true, eSolutionItemToSave solutionIt
                 }
                 if (solutionItemToSave != eSolutionItemToSave.ALMSettings)
                 {
-                    if (this.ALMDomain != lastSavedSolution.ALMDomain || this.ALMProject != lastSavedSolution.ALMProject || this.ALMServerURL != lastSavedSolution.ALMServerURL || this.AlmType != lastSavedSolution.AlmType)
+                    //if (this.ALMDomain != lastSavedSolution.ALMDomain || this.ALMProject != lastSavedSolution.ALMProject || this.ALMServerURL != lastSavedSolution.ALMServerURL || this.AlmType != lastSavedSolution.AlmType)
```

Comment:
Why this condition is commented out ? With this when you save solution xml it will show message that ALM config is modified even if it is not modfied

Reply:
Done please review